modyn/evaluator/internal/metrics/accuracy.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any
+
+import torch
+from modyn.evaluator.internal.metrics.abstract_decomposable_metric import AbstractDecomposableMetric
+
+
+class Accuracy(AbstractDecomposableMetric):
+    """
+    Accuracy metric implementation.
+    """
+
+    def __init__(self, evaluation_transform_func: str, config: dict[str, Any]) -> None:
+        super().__init__(evaluation_transform_func, config)
+        self.seen_samples = 0
+        self.total_correct = 0
+
+    def batch_evaluated_callback(self, y_true: torch.tensor, y_pred: torch.tensor, batch_size: int) -> None:
+        if self.evaluation_transformer_function:
+            y_true, y_pred = self.evaluation_transformer_function(y_true, y_pred)
```

Comment:
If this is something all metrics do, maybe we should have a function in the abstract class that does this, and then calls the AbstractImpl? So evaluator calls function XYZ on abstractclass, that does this, and then calls the specific impl